eng/.docsettings.yml

```diff
@@ -126,6 +126,7 @@ known_content_issues:
   - ['sdk/anomalydetector/Azure.AI.AnomalyDetector/README.md', '#5499']
   - ['sdk/core/Azure.Core/README.md', '#5499']
   - ['sdk/core/Azure.Core.Experimental/README.md', '#5499']
+  - ['sdk/core/System.BinaryData/README.md', '#5499']
```

Comment:
I think the name of the package should be more general. Otherwise we will end up with too many very small depednecies from Azure SDK libraries. I would like to name the package System.Buffers.Data (or something like that) so that we can add more related types to this package in the future.